### PR TITLE
Testmodalbranch

### PR DESCRIPTION
**Pull request by rokasGithub**
Features videos

### CODE REVIEW

File: app/index.pug [hunk 1]
```diff
@@ -637,9 +710,24 @@ html(lang='en')
...
     script.
       window.dataLayer = window.dataLayer || [];
       function gtag(){dataLayer.push(arguments);}
       gtag('js', new Date());
       gtag('config', 'UA-39444052-20', { 'optimize_id': 'GTM-MXXGKXQ'});
+
+    script(type='text/javascript').
+
+      $(document).ready(function(){
+        $('#fontsModal, #rulerModal, #colorsModal, #textSpeechModal, #overlayModal, #explainWordModal').on('hidden.bs.modal', function() {
```
Comment (RobertJGabriel) [inline app/index.pug]: Why not instead of the ideas, use a class then the this. function.

Comment (RobertJGabriel) [inline app/index.pug]: Put it in its own file.

File: app/index.pug [hunk 2]
```diff
@@ -301,6 +358,19 @@ html(lang='en')
                 | With one right click of a mouse, 
                 | users can disover what any word means.
                 | This includes synonyms and related words.
+                br
+                button.btn.btn-info.btn-sm(type='button', data-toggle='modal', data-target='#explainWordModal') Example
```
Comment (RobertJGabriel) [inline app/index.pug]: Example? Maybe "See in action"

File: app/index.pug (hunk 1 above)
Comment (rokasGithub) [reply app/index.pug]: complete and put the crisp code in it to+ comments

**Pull request merged**